src/anyvar/utils/funcs.py

```diff
@@ -19,3 +21,25 @@ def get_nested_key(dict_object: dict, *keys: Any) -> Any:  # noqa: ANN401
         else:
             return None
     return None
+
+
+def get_nested_attribute(class_object: object, *attributes: Any) -> Any:  # noqa: ANN401
+    """Traverses the path of nested `attributes` in `class_object` and returns the final value.
+    If any key does not exist, returns None.
+
+    :param class_object: The class_object to traverse.
+    :param attributes: The attributes that will be used to traverse the attributes of the class_object, in the order they should be used.
+    :returns: The final value in the property traversal, if all attributes exist; else None.
```

Comment:
I think we use return? Idk if it's a big deal, just want to be consistent (looks like there are other places in the codebase that use returns instead of return) 

```suggestion
    :return: The final value in the property traversal, if all attributes exist; else None.
```